src/libs/ui/searchitemdelegate.cpp

```
@@ -195,13 +142,64 @@ void SearchItemDelegate::paint(QPainter *painter, const QStyleOptionViewItem &op
+        const QFontMetrics boldFm(boldFont);
+
+        const QColor matchColor = (opt.state & QStyle::State_Selected)
+                ? opt.palette.color(cg, QPalette::HighlightedText)
+                : opt.palette.color(QPalette::Active, QPalette::Highlight);
+        const QColor textColor = painter->pen().color();
+
+        QSet<int> matchSet(matchPositions.begin(), matchPositions.end());
+
+        // Match positions are indices into the original text. When elided,
+        // stop highlighting before the ellipsis to avoid mismatched indices.
+        const bool isElided = (elidedText != opt.text);
```

Clang (14.0.6)

[warning] variable 'isElided' is not initialized

(cppcoreguidelines-init-variables)